x-pack/plugins/siem/public/management/pages/policy/store/policy_details/selectors.ts

```diff
@@ -31,22 +34,24 @@ export const policyDetailsForUpdate: (
 
 /** Returns a boolean of whether the user is on the policy details page or not */
 export const isOnPolicyDetailsPage = (state: Immutable<PolicyDetailsState>) => {
-  if (state.location) {
-    const pathnameParts = state.location.pathname.split('/');
-    return pathnameParts[1] === 'policy' && pathnameParts[2];
-  } else {
-    return false;
-  }
+  return (
+    matchPath(state.location?.pathname ?? '', {
```

Comment:
We now use React Router `matchPath` to determine if we're on the Policy List/Details pages